packages/design-system-docs/src/example.js

```diff
@@ -31,14 +31,14 @@ function setModalExamples() {
   const dialogHeading = document.getElementById('dialog-heading');
   const dialogWrap = document.getElementById('dialog-wrap');
 
-  document.querySelectorAll('.dialog-open-btn').forEach((btn) => {
+  document.querySelectorAll('.dialog-open-btn').forEach(btn => {
```

Comment:
@line47 A lot of these style changes should not have been included in this PR, as they go against our new linting rules. I'm not sure exactly why they are being included here (maybe your local dev env needs to be updated or reconfigured), but if you run `yarn lint` these changes should be undone.

Reply:
Hmmm yeah that is odd, I'll take a look. Thanks!

Reply:
Looks like these were introduced when I merged master into this branch 
https://github.com/CMSgov/design-system/pull/677/commits/61ebb45f4a799301ffe934ae0d4bdfd5266090b1